c10/util/BFloat16.h

```diff
@@ -43,6 +43,19 @@ namespace detail {
 
     return res >> 16;
   }
+
+  inline C10_HOST_DEVICE uint16_t round_to_nearest_even(float src) {
```

Comment:
Can you, please, add some tests for this?

Reply:
RNE test is added into c10/test/util/bfloat16_test.cpp